simulator/launch/basic_track.launch

```diff
@@ -1,21 +1,37 @@
 <?xml version="1.0" encoding ="UTF-8"?>
 <launch>
+    <arg name="mav_name" default="hummingbird"/>
+    <arg name="enable_logging" default="false" />
+    <arg name="enable_ground_truth" default="true"/>
+    
+    <include file="$(find simulator)/launch/world.launch" />
 
-    <arg name="world" default="$(find simulator)/worlds/blue_frames.world"/>
+    <group ns="$(arg mav_name)">
+        <include file="$(find rotors_gazebo)/launch/spawn_mav.launch">
+          <arg name="mav_name" value="$(arg mav_name)" />
+          <arg name="model" value="$(find simulator)/models/urdf/mav_with_camera.gazebo" />
```

Comment:
keep all the value tags in the same vertical column, this increases readability. you can look at the planning lib launch files for reference.